### PR TITLE
[IMPORTED] Transaction sheet preview (bounty) 

### LINKED ISSUE #16264: Partially implement browser transaction sheet
This issue is to track the implementation of the browser transaction sheet:

https://www.figma.com/file/VsV8jYZc7jYUHkPkOUrkyR/Browser-for-Mobile?type=design&node-id=816-17181&t=QRhLsaP5uVuyZWts-4

Please re-use existing components when available and place implementation in the preview screens.

Only what is in this screenshot is in scope, please use figma as a reference though.

Select account should be scrollable horizontally and should show the other accounts.

You can use mock data for the preview.

Have a word with our designers before staring to clarify interactions.

selector

### PR DESCRIPTION
**Pull request by siddarthkay**
Fixes #16264 

### Summary
Implemented preview for transaction sheet

#### Added:
1. Preview screen for transaction sheet (partial)
2. Locked input component (used in preview screen twice)

#### Fixed:
1. Account selector component to match the design
2. purple 50 colors (to also match the design)

On a previous screen I've re-used components like:
1. account-selector
2. segmented-control

### Steps to test

- Open Status
- Unlock app
- Press the top-left user settings button
- In Settings, press Quo2.0 Preview
- Scroll to "transaction-sheet"
- Slide the account selectors list horizontally
- Ensure that all components are in place

### CODE REVIEW

File: src/quo2/components/tabs/account_selector.cljs [hunk 1]
```diff
@@ -66,7 +69,7 @@
                   :margin-bottom 8}}
         label-text])
      [rn/view {:style (account-container-row background-color)}
-      [rn/view {:style account-emoji-container}
+      [rn/view {:style (account-emoji-container (theme/get-theme))}
```
Comment (J-Son89) [inline src/quo2/components/tabs/account_selector.cljs:72]: an oversight
account emoji component should be used here: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=483-19407&mode=design&t=O9RfvM3T2UYkamL3-4

There is this task here to update this component so either way this needs to be updated at some point here to correctly use this component - https://github.com/status-im/status-mobile/issues/16761

File: src/quo2/core.cljs [hunk 2]
```diff
@@ -187,6 +188,7 @@
 (def recovery-phrase-input quo2.components.inputs.recovery-phrase.view/recovery-phrase-input)
 (def search-input quo2.components.inputs.search-input.view/search-input)
 (def title-input quo2.components.inputs.title-input.view/title-input)
+(def locked-input quo2.components.inputs.locked-input.view/locked-input)
```
Comment (J-Son89) [inline src/quo2/core.cljs:191]: nit: alpha sort 😿

File: src/status_im2/contexts/quo_preview/transaction_sheet/transaction_sheet.cljs [hunk 3]
```diff
@@ -0,0 +1,82 @@
+(ns status-im2.contexts.quo-preview.transaction-sheet.transaction-sheet
```
Comment (J-Son89) [inline src/status_im2/contexts/quo_preview/transaction_sheet/transaction_sheet.cljs:1]: this screen needs to be nuked- and replaced with a preview screen for locked-input

File: src/quo2/foundations/colors.cljs [hunk 4]
```diff
@@ -208,7 +208,7 @@
                60 "#1A4E52"}
    :pink      {50 "#F66F8F"
                60 "#C55972"}
-   :purple    {50 "#7140FD"
+   :purple    {50 "#8661C1"
```
Comment (J-Son89) [inline src/quo2/foundations/colors.cljs:211]: colors need to be verified with foundations - https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Foundations?type=design&node-id=619-9895&mode=dev

File: translations/en.json [hunk 5]
```diff
@@ -738,6 +738,7 @@
     "join-decentralised-communities": "Join Decentralized Communities",
     "http-gateway-error": "Oops, request failed!",
     "sign-request-failed": "Could not sign message",
+    "simple": "Simple",
```
Comment (J-Son89) [inline translations/en.json:741]: I think we can remove the translations as only used in the preview screen that we want to remove

**Review by J-Son89 — CHANGES_REQUESTED** [review]

File: src/quo2/components/tabs/account_selector.cljs (hunk 1 above)
Comment (J-Son89) [reply src/quo2/components/tabs/account_selector.cljs:72]: @smohamedjavid can you fix this in your pr?

Comment (smohamedjavid) [reply src/quo2/components/tabs/account_selector.cljs:72]: Sure thing 👍

**Comment by siddarthkay** [comment]
going to re-import. closing this PR!

**Pull request closed without merging**